qa/rpc-tests/addressindex.py

```diff
@@ -291,6 +307,15 @@ def check_balance(node_index, address, expected_balance, expected_received=None)
             assert_equal(deltas[i]['satoshis'], expected_deltas[i]['satoshis'])
             assert_equal(deltas[i]['txid'],     expected_deltas[i]['txid'])
 
+        # test getaddressdeltas in the lightwalletd node
+        deltas = self.nodes[3].getaddressdeltas({'addresses': [addr1]})
+        assert_equal(len(deltas), len(expected_deltas))
+        for i in range(len(deltas)):
+            assert_equal(deltas[i]['address'],  addr1)
+            assert_equal(deltas[i]['height'],   expected_deltas[i]['height'])
+            assert_equal(deltas[i]['satoshis'], expected_deltas[i]['satoshis'])
+            assert_equal(deltas[i]['txid'],     expected_deltas[i]['txid'])
```

Comment:
To avoid code duplication, I suggest:
```
        # test getaddressdeltas
        for node in (1, 3):
            deltas = self.nodes[node].getaddressdeltas({'addresses': [addr1]})
            assert_equal(len(deltas), len(expected_deltas))
            for i in range(len(deltas)):
                assert_equal(deltas[i]['address'],  addr1)
                assert_equal(deltas[i]['height'],   expected_deltas[i]['height'])
                assert_equal(deltas[i]['satoshis'], expected_deltas[i]['satoshis'])
                assert_equal(deltas[i]['txid'],     expected_deltas[i]['txid'])
```